make_test_images/src/compare_manifests.rs

```diff
@@ -183,7 +183,10 @@ fn compare_json_values(
                 || path.ends_with(".instanceId")
                 || path.ends_with(".time")
                 || path.contains(".hash")
-                || val1.is_string() && val2.is_string() && val1.to_string().contains(":urn:uuid:"))
+                || val1.is_string()
+                    && val2.is_string()
```

Comment:
Is this something we want to test against since it will not be available in 2.1?

Reply:
We will need to update this to handle 2.1 uris.